src/main/kotlin/build/buf/intellij/annotator/BufAnalyzeUtils.kt

```diff
@@ -185,12 +191,25 @@ object BufAnalyzeUtils {
         )
         handler.addProcessListener(object : ProcessAdapter() {
             override fun onTextAvailable(event: ProcessEvent, outputType: com.intellij.openapi.util.Key<*>) {
-                result.add(event.text.trimEnd())
```

Comment:
There are three output types - SYSTEM/STDOUT/STDERR. This change ensures that we don't treat any of the other output types as stdout.